### ISSUE TITLE
"tldr --update" shows message "Local data is older than two weeks, use --update to update it."

### ISSUE THREAD

**Issue by sushsada** [tldr-pages/tldr-cpp-client#38]
## Reproduction Rate
everytime
## Steps to Reproduce
Install tldr.
Wait more than 2 weeks
perform Local data is older than two weeks, perform "tldr --update"

## Result
Shows error "Local data is older than two weeks, use --update to update it."

## Expected Result
Why is it showing me an error to update, when I just passed the update flag?

## Additional Information
~ $tldr --version
tldr v1.3.0 (v1.3.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-cpp-client

**Comment by rodrigobdz** [comment]
It seems to have been fixed but no release was made since then. 
https://github.com/tldr-pages/tldr-cpp-client/blob/0603faf4280af75315408974d091949cf2929ec3/src/tldr.c#L106-L108

The current latest version `1.3.0` does not check if the update flag was passed.
https://github.com/tldr-pages/tldr-cpp-client/blob/1dd60fcf49382702ff9cfac29dd7fa5a2ab82d57/src/tldr.c#L50-L52
@Leandros Could you release a new version with this fix?

**Comment by rodrigobdz** [comment]
I just installed `tldr` with `brew install tldr --HEAD` and I confirm that the bug is fixed in the current HEAD (9b23d268548d3324581997c07df453542b302d51).
![image](https://user-images.githubusercontent.com/14152377/50918371-5c450380-1440-11e9-90ad-8fbf1f7ec11d.png)

**Comment by ortem** [comment]
Still having this redundant warning, @Leandros could you please release a new stable version?
```
~ » tldr --version
tldr v1.3.0 (v1.3.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-cpp-client

~ » tldr --update
Local data is older than two weeks, use --update to update it.
```

**Comment by bl-ue** [comment]
@waldyrious / @sbrl, could we release a new version?

**Comment by sbrl** [comment]
@bl-ue what precisely do you mean by release a new version? I'm happy to add a new GitHub release, but I would assume there would be some package managers that would also need to be updated.

Is there a "making a release" guide anywhere?

**Comment by waldyrious** [comment]
@sbrl I believe the workflow in @owenvoke's #55 will react when a new release is tagged in this repo, and automatically create a PR to update the version of [`tldr` in the Homebrew package manager](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tldr.rb) (which is the only one [mentioned in the README](https://github.com/tldr-pages/tldr-c-client#installing), so I assume no other updates would be needed). Can you merge that PR so we could test that flow?

**Comment by sbrl** [comment]
Oops! This must have been drowned in my GH notifications.

PR merged, @waldyrious!

**Comment by waldyrious** [comment]
No worries @sbrl :) thanks for merging the PR!
